### PR TITLE
empty state for `notification tables` when there are `settings`

### PR DESCRIPTION
**Pull request by marcel-bitfly**
Remove `Click here to set notifications` text for `empty tables`, when there are already `notification settings` for the specifc table

### CODE REVIEW

File: frontend/composables/notifications/useNotificationsManagementDashboards.ts [hunk 1]
```diff
@@ -126,7 +126,7 @@ export function useNotificationsManagementDashboards() {
           dashboard_id,
           group_id,
         },
-      )
+      ).then(() => refreshOverview())
```
Comment (marcel-bitfly) [inline frontend/composables/notifications/useNotificationsManagementDashboards.ts:129]: this wont work as you are returning on line `120`

Comment (benji-bitfly) [reply frontend/composables/notifications/useNotificationsManagementDashboards.ts:129]: ahh of course , thx :pray:

File: frontend/composables/notifications/useNotificationsManagementDashboards.ts [hunk 2]
```diff
@@ -137,7 +137,7 @@ export function useNotificationsManagementDashboards() {
         dashboard_id,
         group_id,
       },
-    )
+    ).then(() => refreshOverview())
```
Comment (marcel-bitfly) [inline frontend/composables/notifications/useNotificationsManagementDashboards.ts:140]: you are still breaking the functionality of `deleteDashboardNotifications` for `validator dashboards` as the `fetch` now returns the value of `refreshOverview()`

Comment (benji-bitfly) [reply frontend/composables/notifications/useNotificationsManagementDashboards.ts:140]: :face_with_open_eyes_and_hand_over_mouth: Thank you for pointing this out, gonna take care of it `asap` :pray:

**Review by marcel-bitfly — COMMENTED** [review]
👌 nice work @benji-bitfly 
thx for taking over

**Pull request merged**